modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1380,18 +1459,23 @@ This command does not take any options, except for the default ones: `--help`, `
 
 This command displays information on the application you pass in `<name>`. +
 You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
-It will return data such as the application's domain, its status, last time it was updated, the Mule version, the ZIP file name, the region, monitoring, and workers; as well as `TRUE` or `FALSE` information for persistent queues and static IPs enablement. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+It will return data such as the application's domain, its status, last time it was updated, the Mule version, the ZIP file name, the region, monitoring, and workers; as well as `TRUE` or `FALSE` information for persistent queues and static IPs enablement. 
```

Comment:
oh, "the last time it was updated"

I'm not sure about "data such as ... monitoring, and workers; as well as ...".

Reply:
Yep!